.gitignore

```
@@ -4,7 +4,7 @@ __pycache__/
 *$py.class
 
 # C extensions
-*.so
+*.so 
```

```suggestion
*.so
```

No need for this change, I guess it is another leftover :)